lib/util/helpers.js

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+var assert = require('assert');
+
+/**
+ * @param {Any} param The property to assert
+ * @param {String} message The failing message
+ * @returns {Error} an Error instance or null
+ */
+exports.doAssert = function(param, message) {
+  try {
+    assert(param, message);
+  } catch (err) {
+    return err;
+  }
+  return null;
+};
```

Comment:
@kjdelisle I haven't changed everywhere that uses `assert` since I am not sure if this is the correct way....
If you define the `try...catch` function inside the main function, you can return callback and stop executing code afterwards.
While if we extract the function to another file, you can not actually return the main function's callback....
So "do try catch everywhere" VS "extract it like what I did in this commit", which one would you choose :-p ?